src/index.js

```diff
@@ -11,12 +11,14 @@ if (!process.env.KV_USERNAME || !process.env.KV_PASSWORD) {
 }
 
 let args = process.argv;
-if (args[0] =~ /node$/) {
+if (args[0].match(/node$/)) {
   args = args.slice(2);
-} else if (args[0] =~ /npm$/) {
+} else if (args[0].match(/npm$/)) {
   args = args.slice(3);
 }
 
+console.log(args);
+
```

Comment:
let's remove the console logs here and below, I'm sure they were helpful in validating this change worked, but it's just noise for end users.

Reply:
My bad.